MarioGame/Content/Levels/Level1.json

```diff
@@ -304,6 +304,41 @@
             "position": {
                 "x": 6047,
                 "y": 544 }
+        },
+        {
+            "type": "FIREBALL",
```

Comment:
fireballs as such are not entities of the level so they should not be in the level json, try to generate them based on an event or key when the level is running.